.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+    - '0.10'
```

Comment:
[aXe Core is only testing 0.10](https://github.com/dequelabs/axe-core/blob/master/.travis.yml).   I could test both and see how it goes.